src/components/HURUmap/Panel/DesktopPanel/RichData.js

```diff
@@ -8,25 +8,19 @@ import TreeView from "@/pesayetu/components/HURUmap/TreeView";
 
 function RichData({ primaryProfile, ...props }) {
   const classes = useStyles(props);
-  const [expanded, setExpanded] = useState();
   const profileRef = useRef();
 
-  const handleLabelClick = (e) => {
-    e.preventDefault();
-    const { id, expand } = e.target.dataset;
-    const { current: el } = profileRef;
-    if (el) {
-      el.querySelector(`#${id}`).scrollIntoView({ behavior: "smooth" });
-      if (expand) {
-        setExpanded(id);
-      }
+  const handleLabelClick = (id) => {
+    const el = profileRef?.current;
+    if (el && id) {
+      document.documentElement.style.scrollBehavior = "smooth";
```

Comment:
@kilemensi `scroll-behavior` work when added on `html`. Anyway, because behavior wasn't actually an issue, I will add it back as option of scrollIntoView

Reply:
Is this another Chrome issue @KhadijaMahanga ? From the [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior), it seem like you should be able to set `scroll-behavior` per `div`, no?